packages/shorebird_cli/lib/src/validators/android_internet_permission_validator.dart

```diff
@@ -67,6 +67,7 @@ class AndroidInternetPermissionValidator extends Validator {
             (String manifestPath) => ValidationIssue(
               severity: ValidationIssueSeverity.error,
               message: '$manifestPath is missing the INTERNET permission.',
+              fix: () async => _addInternetPermissionToFile(manifestPath),
```

Comment:
```suggestion
              fix: () => _addInternetPermissionToFile(manifestPath),
```
does it need to be async?

Reply:
Yes – `fix` is defined as `final Future<void> Function()? fix;`

Reply:
Saw your next comment, will remove